docs/release_flow/linux.md

```diff
@@ -0,0 +1,16 @@
+
+
+# Install Linux Preview-Releases: 
+
+On `promote-client` we will upload .deb packages from taskcluster to the [mozilla apt repo](https://blog.nightly.mozilla.org/2023/10/30/introducing-mozillas-firefox-nightly-deb-packages-for-debian-based-linux-distributions/)
+
+## First, import the APT repository signing key:
+wget -q https://repository.stage.productdelivery.nonprod.webservices.mozgcp.net/apt/repo-signing-key.gpg -O- | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/repository.stage.productdelivery.nonprod.webservices.mozgcp.net.gpg > /dev/null
```

Comment:
This is the dep (staging) repository. When these tasks run on the shipping action they will upload the packages to the production repository. Also, the `trusted.gpg.d` directory is no longer the place to put these third-party repository keys.
```suggestion
sudo install -d -m 0755 /etc/apt/keyrings
wget -q https://packages.mozilla.org/apt/repo-signing-key.gpg -O- | sudo tee /etc/apt/keyrings/packages.mozilla.org.asc > /dev/null
```